argo-cd-apps/base/vault/vault.yml

```diff
@@ -0,0 +1,117 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: vault
+  namespace: openshift-gitops
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+
+spec:
+  project: default
+
+  source:
+    chart: vault
+    repoURL: https://helm.releases.hashicorp.com
+    targetRevision: v0.19.0
+    helm:
+      values: |
+        global:
+          openshift: true
+          tlsDisable: false
+        ui:
+          enabled: true
+        #injector config
+        injector:
+          image:
+            repository: "registry.connect.redhat.com/hashicorp/vault-k8s"
+            tag: "0.15.0-ubi"
+          agentImage:
+            repository: "registry.connect.redhat.com/hashicorp/vault"
+            tag: "1.10.0-ubi"
+          certs:
+            secretName: vault-agent-injector-tls
+            #caBundle: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+            certName: tls.crt
+            keyName: tls.key
+          service:
+            annotations:
+              #utilize OCP service-ca for certs
+              service.beta.openshift.io/serving-cert-secret-name: vault-agent-injector-tls
+        #server config
+        server:
+          image:
+            repository: "registry.connect.redhat.com/hashicorp/vault"
+            tag: "1.10.0-ubi"
+          route:
+            enabled: true
+            host: vault.apps.cluster-k2wgc.k2wgc.sandbox861.opentlc.com
```

Comment:
rhpds url here

Reply:
is there a way to make it refer to something local? if not, we should document updating it

Reply:
Needs additional research to figure out a good solution here. Updated README with info on what to update.